src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        int numberFigure = 4;
```

Comment:
numberFigure is local variable. It should be constant

Reply:
not fixed